<launch>
  <node pkg="uniboard_communication" type="uniboard_service.py" name="uniboard_communication"/>
  <node pkg="rosserial_python" type="serial_node.py" name="rosserial_python" args="/dev/ttyUSB0 _baud:=14400"/>
  <node pkg="flodometry" type="Flodometry.py" name="flodometry"/>
  <node name="test_motion_control" pkg="vel_pid" type="test_real_rover_pid.py" />
</launch>